### DISCUSSION TITLE
喵斯快跑|官方中文|Build.15122534-欲魂魔音+正如计划的那样DLC|解压即撸

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1446] [其他]
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/774171/capsule_616x353.jpg?t=1719367052)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1055810/capsule_616x353.jpg?t=1657533782)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1055810/ss_159747a2eaf44fe0d38aeb1ad69fa807ada96346.600x338.jpg?t=1657533782)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1055810/ss_e1148950cea29943b165ae6456805598a9404fa9.600x338.jpg?t=1657533782)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1055810/ss_0ce20b6eb756ffc407604e3c98e790705dff448f.600x338.jpg?t=1657533782)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1055810/ss_43c52062b06cd948581d47c880e6ed8f99d8b14b.600x338.jpg?t=1657533782)
边听音乐边打。”
“那是来自异世界的电话吗？”
【游戏开始——！！】

可爱的女孩
还是有节奏的音乐？
如果你有一个地方可以同时拥有它们……
一定是跑酷和节奏游戏的天堂 ★★★ — Muse Dash！！

咦？！不是你的强项？没关系！
谁说你必须有很强的节奏感才能获胜？
避免也是一个有效的选项 (?)
只要你对可爱的女孩有100%的热情、勇气和……爱~
节奏和战斗都不是问题ヾ(❛ᴗ˂)♥

 链接: https://pan.baidu.com/s/1zwEWN2QipmdgZM389vgasw

提取码: ij9p

解压密码:laoquzhang.com

**Comment by lllliufeng** [comment]
啊？

**Comment by lllliufeng** [comment]
3大妈开到绳网了是吧（

**Comment by kallen1314** [comment]
啊？这也行？别被举报了

**Reply by kallen1314** [reply]
别连累我家绳网一起被举报
